readme.md

```diff
@@ -83,7 +84,6 @@ Hello World !  % This is your content
 ```
 
 It should look like this in TexMaker:
-![](http://i.imgur.com/ZuD5N6U.png)
```

Comment:
Why?

Reply:
There were extra spaces at the end of the line, it seems I ended up messing up and deleting it, reverting to previous version without the spaces